decidim-admin/app/packs/entrypoints/decidim_admin.js

```diff
@@ -10,6 +10,7 @@ import "src/decidim/admin/newsletters"
 import "src/decidim/admin/form"
 import "src/decidim/admin/external_domain_allowlist"
 import "src/decidim/admin/draggable-list"
+import "src/decidim/admin/draggable-table"
```

Comment:
We have a similar functionality added by #13112. 

Could you remove this, and use the respective API ?

```suggestion
```

Reply:
Hi, sorry for the late response, I'm taking this back today :) 

I've checked https://github.com/decidim/decidim/pull/13112, and it seems it doesn't replace the logic I added in this file. Notice that the new API is just a way to define the `sortable` options via markup; it doesn't replace managing the logic when someone drags and drops. For example, that logic is defined [here](https://github.com/decidim/decidim/pull/13112/files#diff-61248ecb512972c136c5b5407a23a318b1e0b1301b059042df95a60c55f2a832R32-R70) in the PR you mentioned. In my case, that logic is defined in this `draggable-table.js` file. 

Also, I'm using the [sort_list.component](https://github.com/decidim/decidim/blob/develop/decidim-admin/app/packs/src/decidim/admin/sort_list.component.js), to simplify defining that logic, which is used in six other places.

Reply:
Also while playing with the code i have noticed the api is different. I will have a new look on this PR, and what is being implemented in the taxonomies, and will keep you updated.

Reply:
I'll take a look at the look and feel and reuse as much as possible.

Reply:
@alecslupu I see 3 main differences in right vs left (us). From a design POV I don't see any reason for any of the changes: 

1.- lack of vertical lines: are we removing them anywhere else? Any criteria?
2.- style for cursor hove over the reorder icon: I think the one we are using is more appropiate
3.- padding for the item: again, this seems something new we are not doing anywhere else.

Confirm if we should apply all three.

Reply:
@furilo: 
1. Lack of vertical lines - This needs to be fixed in Taxonomy table ( i will handle this ) 
2. Style of the cursor is OK as per your implementation. 
3. Padding / Size / Colors should be aligned with the Taxonomies. 

My issue when i have initially reviewed this PR were caused by the fact that your implementation: generated the following:
```
<tbody class="draggable-table" data-sort-url="/admin/participatory_processes/exaggerate-horizon/components/reorder" data-draggable-table="" aria-dropeffect="move">
```
While the taxonomy had the following
```
<tbody class="js-sortable js-draggable-root" data-draggable-handle=".js-drag-handle" data-draggable-accept-from=".js-draggable-root" data-sort-url="/admin/taxonomies/reorder" role="rowgroup" aria-dropeffect="move">
```

My conclusion is that we have 2 ways of implementing draggable tables: once using `data-draggable-table` and the one which is used by taxonomy implementation. 

Now, i have understood better what are the issues and to be honest, i see some changes to be done here, and some in the taxonomies so that we fully align those functionalities. I will have a look on it so that I fully get the picture. 

Cc @microstudi , @andreslucena.

Reply:
Ok, i had a deeper look in this issue, and i think i have found a way that would help us move this forward. I have applied the changes as a standalone PR in #13355 . 

One final round changes from my end, and I can say we are ok to merge this one.